docs/conf.py

```diff
@@ -135,8 +135,9 @@
     "litestar.template": {"litestar.template.base.T_co"},
     "litestar.openapi.OpenAPIController.security": {"SecurityRequirement"},
     re.compile(r"litestar\.plugins.*"): re.compile(".*ModelT"),
+    re.compile(r"litestar\.contrib\.*"): re.compile(".*DataT"),
     re.compile(r"litestar\.contrib\.sqlalchemy\.*"): re.compile(
-        ".*(ConnectionT|EngineT|SessionT|SessionMakerT|SlotsBase|DataT)"
+        ".*(ConnectionT|EngineT|SessionT|SessionMakerT|SlotsBase)"
```

Comment:
I would frankly just rename all of these to T and be done with it

Reply:
OK